tools/manifest-templator/manifest-templator.go

```diff
@@ -443,11 +468,24 @@ func main() {
 	namespace := flag.String("namespace", "kubevirt-hyperconverged", "")
 	csvVersion := flag.String("csv-version", "0.0.1", "")
 	containerPrefix := flag.String("container-prefix", "kubevirt", "")
+	hcoContainerPrefix := flag.String("hco-container-prefix", *containerPrefix, "")
+	kubevirtContainerPrefix := flag.String("kubevirt-container-prefix", *containerPrefix, "")
+	cdiContainerPrefix := flag.String("cdi-container-prefix", *containerPrefix, "")
 	cnaContainerPrefix := flag.String("cna-container-prefix", *containerPrefix, "")
 	webuiContainerPrefix := flag.String("webui-container-prefix", *containerPrefix, "")
+	sspContainerPrefix := flag.String("ssp-container-prefix", *containerPrefix, "")
+	nmoContainerPrefix := flag.String("nmo-container-prefix", *containerPrefix, "")
 	imagePullPolicy := flag.String("image-pull-policy", "IfNotPresent", "")
 	inputFile := flag.String("input-file", "", "")
 
+	cnaMultusImage := flag.String("cna-multus-image", "", "")
```

Comment:
I think we should at least converse about the recent explosion of parameters to this.